src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -214,15 +214,6 @@ protected override async Task<bool> AcceptEvent(SessionId sessionId, JObject par
                 case "Debugger.scriptParsed":
```

Comment:
If we are not doing anything for this, and simply passing it on, then do we need this at all? If we don't have this block then it would be the same behavior, minus the ` logger.LogTrace($"proxying Debugger.scriptParsed ({sessionId.sessionId}) {url} {args}");`.
@thaystg does having the log message sound useful for debugging?

Reply:
I don't think we need the log message.

Reply:
And it would be getting logged already as a protocol message.

Reply:
Checked what @radical suggested offline and indeed, we had log duplicates because of this line.